From dfc7e19dda7a7d5f299b612e6182c6a700b53986 Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin.jansa@lge.com>
Date: Thu, 14 May 2020 19:58:23 -0700
Subject: [PATCH] CMakeLists.txt: add missing tf to CATKIN_DEPENDS

* fixes moveit-ros-warehouse issue:
/jenkins/mjansa/build/ros/ros1-melodic-thud/tmp-glibc/work/aarch64-oe-linux/moveit-ros-warehouse/1.0.2-1-r0/recipe-sysroot-native/usr/bin/aarch64-oe-linux/../../libexec/aarch64-oe-linux/gcc/aarch64-oe-linux/8.2.0/ld: warning: libtf.so, needed by /jenkins/mjansa/build/ros/ros1-melodic-thud/tmp-glibc/work/aarch64-oe-linux/moveit-ros-warehouse/1.0.2-1-r0/recipe-sysroot/opt/ros/melodic/lib/libwarehouse_ros.so, not found (try using -rpath or -rpath-link)
/jenkins/mjansa/build/ros/ros1-melodic-thud/tmp-glibc/work/aarch64-oe-linux/moveit-ros-warehouse/1.0.2-1-r0/recipe-sysroot-native/usr/bin/aarch64-oe-linux/../../libexec/aarch64-oe-linux/gcc/aarch64-oe-linux/8.2.0/ld: /jenkins/mjansa/build/ros/ros1-melodic-thud/tmp-glibc/work/aarch64-oe-linux/moveit-ros-warehouse/1.0.2-1-r0/recipe-sysroot/opt/ros/melodic/lib/libwarehouse_ros.so: undefined reference to `tf::Transformer::Transformer(bool, ros::Duration)'
/jenkins/mjansa/build/ros/ros1-melodic-thud/tmp-glibc/work/aarch64-oe-linux/moveit-ros-warehouse/1.0.2-1-r0/recipe-sysroot-native/usr/bin/aarch64-oe-linux/../../libexec/aarch64-oe-linux/gcc/aarch64-oe-linux/8.2.0/ld: /jenkins/mjansa/build/ros/ros1-melodic-thud/tmp-glibc/work/aarch64-oe-linux/moveit-ros-warehouse/1.0.2-1-r0/recipe-sysroot/opt/ros/melodic/lib/libwarehouse_ros.so: undefined reference to `tf::Transformer::~Transformer()'
/jenkins/mjansa/build/ros/ros1-melodic-thud/tmp-glibc/work/aarch64-oe-linux/moveit-ros-warehouse/1.0.2-1-r0/recipe-sysroot-native/usr/bin/aarch64-oe-linux/../../libexec/aarch64-oe-linux/gcc/aarch64-oe-linux/8.2.0/ld: /jenkins/mjansa/build/ros/ros1-melodic-thud/tmp-glibc/work/aarch64-oe-linux/moveit-ros-warehouse/1.0.2-1-r0/recipe-sysroot/opt/ros/melodic/lib/libwarehouse_ros.so: undefined reference to `tf::Transformer::setTransform(tf::StampedTransform const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/jenkins/mjansa/build/ros/ros1-melodic-thud/tmp-glibc/work/aarch64-oe-linux/moveit-ros-warehouse/1.0.2-1-r0/recipe-sysroot-native/usr/bin/aarch64-oe-linux/../../libexec/aarch64-oe-linux/gcc/aarch64-oe-linux/8.2.0/ld: /jenkins/mjansa/build/ros/ros1-melodic-thud/tmp-glibc/work/aarch64-oe-linux/moveit-ros-warehouse/1.0.2-1-r0/recipe-sysroot/opt/ros/melodic/lib/libwarehouse_ros.so: undefined reference to `tf::Transformer::lookupTransform(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, ros::Time const&, tf::StampedTransform&) const'
collect2: error: ld returned 1 exit status

Upstream-Status: Pending

Signed-off-by: Martin Jansa <martin.jansa@lge.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2171881..0006491 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,7 +16,7 @@ find_package(Boost COMPONENTS system filesystem thread)
 catkin_package(
   INCLUDE_DIRS include
   LIBRARIES warehouse_ros
-  CATKIN_DEPENDS roscpp rostime std_msgs geometry_msgs
+  CATKIN_DEPENDS roscpp rostime std_msgs geometry_msgs tf
   DEPENDS Boost
 )
 
